.github/release-please-config.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+    "changelog-sections": [
+        {
+            "type": "feat",
+            "section": "Features",
+            "hidden": false
+        },
+        {
+            "type": "fix",
+            "section": "Bug Fixes",
+            "hidden": false
+        }
+    ],
+    "packages": {
+        "apps/one_piece_commanded": {
+            "component": "one_piece_commanded"
+        },
+        "apps/one_piece_graceful_shutdown": {
+            "component": "one_piece_graceful_shutdown"
+        },
+        "apps/one_piece_result": {
+            "component": "one_piece_result"
+        },
+        "apps/trogon_error": {
+            "component": "trogon_error"
+        }
+    },
+    "plugins": [
+        {
+            "type": "sentence-case"
+        }
+    ],
+    "tag-separator": "@",
+    "include-v-in-tag": true,
+    "include-component-in-tag": true,
```

Comment:
The tags on this release use `component@v0.0.0` so we need the `tag-separator` configured for `@` and we need to include `v` in the tag.